skins/moono-lexicon/mainui.css

```diff
@@ -202,3 +202,7 @@ Special outer level classes used in this file:
 .cke_editable:focus {
 	background-color: #f0f5ff; /* $primary-l3 */
 }
+
+.cke_editable img {
+	margin: 24px;
```

Comment:
Hey @mateomustapic did you see https://github.com/liferay/liferay-ckeditor/pull/116#issuecomment-673318822 ?

Other than that, I feel that this is not the solution we're looking for. Adding `margin: 24px` we're adding a margin in all the sides of the image but I think that what we want is to only add a margin to the left or to the right side depending on the selected alignment. That's why I was asking what was the previous behavior.

About the pr and the commit messages they say "add padding to images" when we're actually adding a margin ;)

Reply:
hi @carloslancha 
I missed the comment you mentioned, sorry, I will now take a look at the behaviour before applying `moono-lexicon` skin.

Regarding the commit message, I will fix it, rushing mistake :)